core/src/main/java/io/questdb/griffin/FunctionParser.java

```diff
@@ -713,7 +713,7 @@ private Function createFunction(
 
                 if (factory.isWindow()) {
                     // prefer window functions in window context, otherwise non-window functions
-                    sigArgTypeScore += isWindowContext ? -1 : 1;
```

Comment:
I increased the value because function parser kept choosing regular SumLong..GroupByFunction over SumDoubleWindow for expressions like  `sum(1)`.